op-devstack/dsl/fb_ws_proxy.go

```diff
@@ -41,29 +42,55 @@ func (c *FlashblocksWebsocketProxy) Escape() stack.FlashblocksWebsocketProxy {
 }
 
 func (c *FlashblocksWebsocketProxy) ListenFor(logger log.Logger, duration time.Duration, output chan<- []byte, done chan<- struct{}) error {
-	return websocketListenFor(logger, c.Escape().WsUrl(), duration, output, done)
+	wsURL := c.Escape().WsUrl()
+	headers := c.Escape().WsHeaders()
+	return websocketListenFor(logger, wsURL, headers, duration, output, done)
 }
 
-func websocketListenFor(logger log.Logger, wsURL string, duration time.Duration, output chan<- []byte, done chan<- struct{}) error {
+// TODO(#986): Revamp this function so that it's more robust,
```

Comment:
@serpixel I'm pretty sure this isn't the issue you meant to reference as it's been closed for quite a while now. It's causing the todo check job to fail: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/99061/workflows/efc17fd8-4285-42a6-8aa6-dfe880f6009e/jobs/3805476

Is this TODO actually necessary? Seems like it should just be removed and either the revamp will happen when its required or there should be a GitHub issue to track it.